static/monorail.yml

```diff
@@ -1305,6 +1305,102 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  /nodes/{identifier}/tags:
+    get:
+      summary: |
+        get the tags associated with a node.
+      description: |
+        get the tags associated with a node.
+      parameters:
+        - name: identifier
+          in: path
+          description: |
+            Mac addresses and unique aliases to identify the node by
+          required: true
+          type: string
+      tags:
+        - nodes
+        - tags
+        - get
+      responses:
+        200:
+          description: tags
+          schema:
+            type: object
```

Comment:
I need to fix this part. I submitted this patch before #292, where I looked for arrays vs objects.  Just double-checked, and this response comes back as an array of strings:

```
[travis@mbp ~/git/RackHD/on-http/static (master)]$ curl http://localhost:9090/api/1.1/nodes/573cb71ed11f55c1075ec878/tags | python -m json.tool
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100     2  100     2    0     0    148      0 --:--:-- --:--:-- --:--:--   166
[]
[travis@mbp ~/git/RackHD/on-http/static (master)]$ curl http://localhost:9090/api/1.1/nodes/5727b4420a899379066c8d05/tags | python -m json.tool
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100     7  100     7    0     0    548      0 --:--:-- --:--:-- --:--:--   583
[
    "k8s"
]
```